src/main/webapp/creatives/maps/simid_map_creative.test.js

```diff
@@ -63,3 +72,32 @@ test('testing button text updates with default ad params', () => {
     expect(button.innerText).toBe('Find Nearest Location');
 });
 
+test('instance of map is instantiated on button click', () => {
+    const eventData = {
+        args: {
+            creativeData: {
+                adParameters: '{}',
+            },
+            environmentData: {},
+        },
+        messageId: 0,
+        sessionId: "test-session-id",
+        timestamp: 0,
+        type: "SIMID:Player:init",
+    };
```

Comment:
This looks like it gets repeated in a few tests, maybe can be extracted into a helper function of some kind?